### PR TITLE
Version 2.02 Dependabot and test output

### PR DESCRIPTION
**Pull request by akmiller01**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  # Enable version updates for GitHub Actions
+  - package-ecosystem: 'github-actions'
+    # or GitHub Actions, set the directory to / to check for workflow files in .github/workflows.
+    directory: '/'
```
Comment (nosvalds) [inline .github/dependabot.yml:6]: I think we'll want `target-branch` for this as well?

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
-          python-version: ${{ matrix.python-version }}
+          python-version: '3.9'
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get -y install libxml2-dev libxslt-dev libxslt1-dev python3-dev python3-lxml libxml2-utils
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install -r requirements.txt
 
       - uses: actions/cache@v2
```
Comment (nosvalds) [inline .github/workflows/CI.yml]: Should be able to remove this and use the built-in feature of `actions/setup-python` 
https://github.com/actions/setup-python#caching-packages-dependencies

**Review by nosvalds — CHANGES_REQUESTED** [review]

File: .github/workflows/CI.yml (hunk 2 above)
Comment (akmiller01) [reply .github/workflows/CI.yml]: I can move the pip parts into setup-python, but the libxml and libxslt dependencies will need to be apt-get installed prior to it. Without those trying to pip install lxml always fails.

Comment (nosvalds) [reply .github/workflows/CI.yml]: Ohh shoot my comment got added in the wrong place. I meant it to be on the `- uses: actions/cache@v2` line!

Comment (akmiller01) [reply .github/workflows/CI.yml]: Okay good point. So far IATI-Codelists was the only repo with a relatively robust GH actions yml, so I carried forward all the previous caching. But you're right we can simplify it by moving the cache from actions/cache to setup-python cache.

**Review by nosvalds — APPROVED** [review]

**Pull request merged**